src/main.ts

```diff
@@ -14,35 +14,35 @@ async function run() {
 
     console.log(`Preparing to sign key @ ${releaseDir} with signing key`);
 
-    // 1. Find release file
-    const releaseFile = io.findReleaseFile(releaseDir);
-    if (releaseFile !== undefined) {
-      core.debug(`Found release to sign: ${releaseFile.name}`);
-
-      // 3. Now that we have a release file, decode and save the signing key
+    // 1. Find release files
+    const releaseFiles = io.findReleaseFile(releaseDir);
+    if (releaseFiles !== undefined) {
+      // 3. Now that we have a release files, decode and save the signing key
       const signingKey = path.join(releaseDir, 'signingKey.jks');
       fs.writeFileSync(signingKey, signingKeyBase64, 'base64');
 
-      // 4. Now zipalign the release file
-      const releaseFilePath = path.join(releaseDir, releaseFile.name);
-      let signedReleaseFile = '';
-      if (releaseFile.name.endsWith('.apk')) {
-        signedReleaseFile = await signApkFile(releaseFilePath, signingKey, alias, keyStorePassword, keyPassword);
-      } else if (releaseFile.name.endsWith('.aab')) {
-        signedReleaseFile = await signAabFile(releaseFilePath, signingKey, alias, keyStorePassword, keyPassword);
-      } else {
-        core.error('No valid release file to sign, abort.');
-        core.setFailed('No valid release file to sign.');
+      // 4. Now zipalign and sign each one of the the release files
+      for (let releaseFile of releaseFiles) {
+        core.debug(`Found release to sign: ${releaseFile.name}`);
+        const releaseFilePath = path.join(releaseDir, releaseFile.name);
+        let signedReleaseFile = '';
+        if (releaseFile.name.endsWith('.apk')) {
+          signedReleaseFile = await signApkFile(releaseFilePath, signingKey, alias, keyStorePassword, keyPassword);
+        } else if (releaseFile.name.endsWith('.aab')) {
+          signedReleaseFile = await signAabFile(releaseFilePath, signingKey, alias, keyStorePassword, keyPassword);
+        } else {
+          core.error('No valid release file to sign, abort.');
+          core.setFailed('No valid release file to sign.');
+        }
+
+        core.debug('Release signed! Setting outputs.');
+        core.exportVariable("SIGNED_RELEASE_FILE", signedReleaseFile);
```

Comment:
Since we are now signing multiple files each file should have its own output/export. So when the length of releaseFiles is > 1, the exports/outputs should be like:

```typescript
core.exportVariable(`SIGNED_RELEASE_FILE_${index}`, signedReleaseFile);
core.setOutput(`SIGNED_RELEASE_FILE_${index}`, signedReleaseFile);
```

However, when the releaseFiles length === 1 then you should only export the above to maintain compatibility, i.e.

```typescript
core.exportVariable("SIGNED_RELEASE_FILE", signedReleaseFile);
core.setOutput("SIGNED_RELEASE_FILE", signedReleaseFile);
```

Once you have done this, you should also update `actions.yml` and `README.md` files to reflect this output.